modules/domUtils.js

```diff
@@ -0,0 +1,27 @@
+// import { showContent } from './domUtils.js';
+
+/* eslint-disable */
```

Comment:
- [ ] As pointed out earlier, please do not disable linters. It helps you better your code quality.